cmd/tidelift-sbom-vulnerability-reporter/main.go

```diff
@@ -20,9 +20,9 @@ import (
 type RecommendationDetails struct {
 	RealIssue             bool    `json:"real_issue"`
 	FalsePositiveReason   *string `json:"false_positive_reason"`
-	ImpactScore           int     `json:"impact_score"`
-	ImpactDescription     string  `json:"impact_description"`
-	WorkaroundAvailable   bool    `json:"workaround_available"`
+	ImpactScore           *int    `json:"impact_score"`
+	ImpactDescription     *string `json:"impact_description"`
+	WorkaroundAvailable   *bool   `json:"workaround_available"`
 	WorkaroundDescription *string `json:"workaround_description"`
 }
```

Comment:
for reference, the generated fields: https://github.com/tidelift/sbom-to-api-tools/blob/main/internal/tidelift-api-gen/tidelift.gen.go#L901-L930